R/plot.R

```diff
@@ -101,49 +104,67 @@
 #'     mapping = aes(x = group, y = group_mean), data = group_means_df,
 #'     colour = 'red', size = 3
 #'   )
-ggplot <- function(data = NULL, mapping = aes(), ...,
```

Comment:
Hmmm, this a weird generic. If I was to write it today, I'd probably call it `as_ggplot`, since it's really a coercion function not a constructor. So it might be better to keep the form of this as is, and instead call the S7 constructor inside of `ggplot.default`.  To avoid name clashes, could call it `S7_ggplot` or similar? Does that make sense?